doc/modules/mixture.rst

```diff
@@ -68,8 +68,9 @@ full covariance.
     * See :ref:`sphx_glr_auto_examples_mixture_plot_gmm_pdf.py` for an example on plotting the
       density estimation.
 
-Pros and cons of class :class:`GaussianMixture`
------------------------------------------------
+|details-start|
+**Pros and cons of class :class:`GaussianMixture`**
```

Comment:
Unfortunately dropdowns break the `:class:` directive. I would suggest to keep the title as plain text i.e.
```suggestion
**Pros and cons of class GaussianMixture**
```

Reply:
Sure, will revert this. Is there a possible fix being discussed for this issue?